json_schema/api/operation.yaml

```diff
@@ -0,0 +1,32 @@
+Operation:
+  description: The status of a long running operation.
+  properties:
+    path:
+      type: string
+      description:
+        The server-assigned path, which is only unique within the same
+        service that originally returns it.
+    done:
+      type: boolean
+      description: >-
+        If the value is false, it means the operation is still in progress.
+        If true, the operation is completed, and either response or error
+        is available.
+    error:
```

Comment:
theoretically, errors should use problem details (RFC 9457): https://aep.dev/193/.

This is the result of an incongreuence  of the proto representation of the response, so grpc-json transcoding would need something custom to actually translate between the proto / grpc convention and the HTTP one.